packages/imt.sol/contracts/internal/InternalLazyIMT.sol

```diff
@@ -157,6 +157,74 @@ library InternalLazyIMT {
         return _root(self, numberOfLeaves, depth);
     }
 
+    function _merkleProofElements(
```

Comment:
Please don't put this function between the `_root` overloads. This function can be either above or below the 3 `_root` functions.

Reply:
done 9e27c98ef556e3bf4c30e7ae716d126f274cd620